src/validation/add.trust.validation.ts

```diff
@@ -1,7 +1,38 @@
+import { Request } from "express";
 import { body } from "express-validator";
 
 import { ErrorMessages } from "./error.messages";
+import { hasNoBoAssignableToTrust } from "../utils/trusts";
+import { getApplicationData } from "../utils/application.data";
+import { ApplicationData } from "../model";
+import { checkRelevantPeriod } from "../utils/relevant.period";
+import { isActiveFeature } from "../utils/feature.flag";
+import { FEATURE_FLAG_ENABLE_RELEVANT_PERIOD } from "../config";
+
+const isAddTrustToBeValidated = (req: Request): boolean => {
+  const appData: ApplicationData = getApplicationData(req.session);
+  const isUpdateOrRemoveJourney: boolean = !!appData.entity_number;
+
+  if (!isUpdateOrRemoveJourney) {
+    return true;
+  }
+
+  if (!hasNoBoAssignableToTrust(appData)) {
```

Comment:
Suggest OR ing this condition with the one on line 16 as return value is `true` in both cases.